scripts/processsourcefiles.sh

```diff
@@ -43,12 +47,13 @@ then
     #git clone -b "release-[VERSION#]" https://github.com/"$FORK"/docs.git temp/release/[VERSION#]
     #mv temp/release/[VERSION#]/docs content/en/[VERSION#]-docs
     ###############################################################
-  git clone --quiet -b "release-0.12" https://github.com/"$FORK"/docs.git temp/release/v0.12
-  mv temp/release/v0.12/docs content/en/v0.12-docs
-  git clone --quiet -b "release-0.11" https://github.com/"$FORK"/docs.git temp/release/v0.11
-  mv temp/release/v0.11/docs content/en/v0.11-docs
-  git clone --quiet -b "release-0.10" https://github.com/"$FORK"/docs.git temp/release/v0.10
-  mv temp/release/v0.10/docs content/en/v0.10-docs
+  mkdir -p docs  # -p won't fail if the file exists
+  for r in $(seq 13 10); do
+    CLONE="docs/release-0.${r}"
```

Comment:
Should this be a variable too for that day when Knative graduates to a MAJOR version?

Reply:
When that happens, I hope to rewrite more of this anyway.

Reply:
ive changed this to use the values set in https://github.com/knative/website/blob/master/scripts/docs-version-settings.sh (so on release time, this file does not need to get updated to include the recent version number)